packages/wallet/src/services/TransactionsTracker.ts

```diff
@@ -148,26 +140,8 @@ export const createTransactionsTracker = (
   }: TransactionsTrackerInternals = {}
 ): TransactionsTracker => {
   const historicalTransactions$ = createHistoricalTransactionsTrackerSubject(transactionsSource$);
-  const providerTransactionsByDirection$ = (inspector: SendReceiveValueInspector) =>
-    combineLatest([historicalTransactions$, addresses$]).pipe(
-      map(([transactions, addresses]) => {
-        const inspectTx = createTxInspector({
-          value: inspector(addresses)
-        });
-        return transactions.filter((tx) => inspectTx(tx).value.coins > 0n);
-      }),
-      distinctUntilChanged(transactionsEquals)
-    );
-
-  const incomingTransactionHistory$ = new TrackerSubject<Cardano.TxAlonzo[]>(
-    providerTransactionsByDirection$(valueReceivedInspector)
-  );
-  const outgoingTransactionHistory$ = new TrackerSubject<Cardano.TxAlonzo[]>(
-    providerTransactionsByDirection$(valueSentInspector)
-  );
-
   const txConfirmed$ = (tx: Cardano.NewTxAlonzo) =>
-    newTransactions$(outgoingTransactionHistory$).pipe(
+    newTransactions$(historicalTransactions$).pipe(
```

Comment:
Does this change make sense?

`txConfirmed$` is being used for most things here:
```ts
outgoing: {
  confirmed$,
  failed$,
  inFlight$,
  pending$,
  submitting$
},
```

Reply:
Yep, looks good